openfga/v1/authzmodel.proto

```diff
@@ -80,10 +82,31 @@ message RelationTypeInfo {
 
 message Metadata {
   map<string, RelationMetadata> relations = 1;
+
+  string module = 2 [(validate.rules).string = {
+    pattern: "^[^:#@\\s]{1,50}$",
+    ignore_empty: true
+  }];
+
+  SourceInfo source_info = 3 [json_name = "source_info"];
+}
+
+message SourceInfo {
+  string file = 1 [(validate.rules).string = {
+    pattern: "\\.fga$",
```

Comment:
we should probably put a character limit on this

Reply:
Yeah I had meant to question the what folks thought here, I'll fix this and validate. I think given the potential for folder structures etc. I'll go with a max length of 100

Reply:
Fixed in 0c609904ae242a60da08ff655b4b1541e736dcc6